### ISSUE TITLE
About initial learning rate

### ISSUE THREAD

**Issue by lld533** [prlz77/ResNeXt.pytorch#15]
Hi,

May I know what's the initial learning rate used in Cifar10 and Cifar100 experiments (-b 128 on 2 GPU cards)? The default value 0.1 or the sample value 0.05? Many thanks in advance!

**Comment by prlz77** [comment]
Hi!

Following the original LUA implementation, for BS=128, the learning rate should be 0.1, since they use lr=0.05 for BS=64 and 0.025 for BS=32.

**Comment by lld533** [comment]
> Hi!
> Following the original LUA implementation, for BS=128, the learning rate should be 0.1, since they use lr=0.05 for BS=64 and 0.025 for BS=32.

Thanks!
